posthog/models/test/test_person_model.py

```diff
@@ -68,3 +78,43 @@ def test_person_is_identified(self):
         person_anonymous = Person.objects.create(team=self.team)
         self.assertEqual(person_identified.is_identified, True)
         self.assertEqual(person_anonymous.is_identified, False)
+
+    def test_delete_person(self):
+        person = Person.objects.create(team=self.team)
+        delete_person(person)
+        ch_persons = sync_execute(
+            "SELECT version, is_deleted, properties FROM person FINAL WHERE team_id = %(team_id)s and id = %(uuid)s",
+            {"team_id": self.team.pk, "uuid": person.uuid},
+        )
+        self.assertEqual(ch_persons, [(100, 1, "{}")])
+
+    def test_delete_person_and_ids(self):
+        person = Person.objects.create(team=self.team, distinct_ids=["distinct_id1"])
+
+        def get_ch_distinct_ids():
```

Comment:
Testing nit: Please extract into a separate class method. We don't really use inline functions this way in python.

Reply:
hmm I disagree here, because this was function is very specific to this test and this test only, used only to prevent repetition but keeping all the context there. 

calling `sync_execute` directly twice now, but I don't see why this would have been an issue